README.md

```diff
@@ -1 +1,52 @@
-# Meals
\ No newline at end of file
+# MEALS KING APP
+A JavaScript Project Meals King App: A web app that display meals categories, category list items, meal items popup model showing details and comment form. The app allows users to like and comment on a favorite meal.
+Meals data is fetched from MealDB API
+Likes and comment is store in Involvement API provided by Microverse.
+
+## Built With
+Webpack
+Loadash
+HTML
+CSS
+JavaScript
+Meal DB API
+Involvement API
+
+## Live Demo (if available)
+Live Demo Link
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+- Go to the repository page. 
+- Press the "Code" button and copy the link. 
+- Clone it using git command git clone.
+- cd into the cloned folder 
+- Run npm install to install all dependancies 
+- Open folder in your code editor 
+- Run npm start to open the project in localhost
+
+## Authors
+👤 Husseini Mudi
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+
+👤 Mohamed Jadib
+GitHub: @jadibdev
+Twitter: @MohamedJadib
+LinkedIn: LinkedIn
```

Comment:
Kindly provide destination links:
Github: https://github.com/jadibdev
Twitter: https://twitter.com/MohamedJadib
LinkedIn: https://www.linkedin.com/in/mohamed-jadib-942a5041/